src/routes/Tool/Goals.tsx

```diff
@@ -269,6 +270,8 @@ const ToolTipContent = () => {
 const Goals = () => {
   const { t } = useTranslation();
   const toolStore = useToolStore();
+  const { sm, md } = useMediaQueries();
+  const mobileInfoIcon = (!sm || !md) && <MdInfoOutline size={24} />;
```

Comment:
To what type does this evaluate? Boolean? I'd rather use a ternary to be safe: `!sm || !md ? <MdInfoOutline size={24} /> : null`